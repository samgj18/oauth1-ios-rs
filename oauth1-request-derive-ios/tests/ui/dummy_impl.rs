#[derive(oauth1_request_ios::Request)]
struct Dummy {
    #[oauth1(unknown)]
    _field: (),
}

impl Dummy where Dummy: oauth1_request_ios::Request {} // OK

fn main() {}
