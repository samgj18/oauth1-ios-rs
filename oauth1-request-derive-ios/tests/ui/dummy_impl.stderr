error: unknown attribute `unknown`
 --> tests/ui/dummy_impl.rs:3:14
  |
3 |     #[oauth1(unknown)]
  |              ^^^^^^^
